### PR TITLE
Update contestants.json

### PR DESCRIPTION
**Pull request by KomalBhadani27**
Add me as a participant!

### CODE REVIEW

File: contestants.json [hunk 1]
```diff
@@ -221,23 +221,16 @@
...
          "github":"https://github.com/strikertanmay",
          "twitter":""
-       },
-	{
-	 "name":"Pulkit Goel",
-         "imageurl":"contestant/pulkit.jpeg",
-         "about":"Yup, that's my profile pic",
-         "facebook":"",
-	 "github":"https://github.com/Tachyon01",
-	 "twitter":""
```
Comment (pradeepgangwar) [inline contestants.json]: Why deleting this record?

File: contestants.json [hunk 2]
```diff
@@ -221,23 +221,16 @@
          "about":"Just a Beginner. but Beware!...",
          "facebook":"https://www.facebook.com/technoamritesh",
          "github":"https://github.com/amriteshbhaskar",
-         "twitter":""	   
+         "twitter":""
+
```
Comment (pradeepgangwar) [inline contestants.json]: Why adding extra line.

**Comment by pradeepgangwar** [comment]
Title and description of PR should be appropriate. Like  `Adding <your-name> to list of contestants`

**Comment by pradeepgangwar** [comment]
Close previous PR when you make new one. Closing this as work continued in #107

**Comment by KomalBhadani27** [comment]
Okay. Actually I am not able to upload my pic as It is showing that uploads
are disabled.

**Comment by pradeepgangwar** [comment]
@KomalBhadani27 Comment here #107

**Comment by KomalBhadani27** [comment]
#107

**Comment by pradeepgangwar** [comment]
@KomalBhadani27 I mean comment your issues/problems on your new PR. This has been closed as you made a new PR which is #107

**Comment by KomalBhadani27** [comment]
Ohh..  Okay.

**Pull request closed without merging**